cookbooks/bcpc-hadoop/templates/default/flume_flume-conf.erb

```diff
@@ -16,3 +16,5 @@
 <%= @agent_name %>.sinks.hdfsSink.hdfs.rollInterval = <%= node['bcpc']['hadoop']['copylog_rollup_interval'] %>
 <%= @agent_name %>.sinks.hdfsSink.hdfs.rollSize = 0
 <%= @agent_name %>.sinks.hdfsSink.hdfs.rollCount = 0
+<%= @agent_name %>.sinks.hdfsSink.hdfs.kerberosPrincipal = <%= node['bcpc']['hadoop']['kerberos']['data']['flume']['principal'] %>/<%= float_host(node[:fqdn]) %>@<%= node['bcpc']['hadoop']['kerberos']['realm'] %>
```

Comment:
This should include the logic to specifiy principal if a particular host is specified.